.circleci/config.yml

```diff
@@ -144,6 +147,9 @@ jobs:
       - run:
           name: Test << parameters.path >>
           command: xcodebuild test -workspace << parameters.workspace >>.xcworkspace -scheme << parameters.scheme >> -sdk iphonesimulator -destination "${destination}" | xcpretty --simple --color --report junit
+      - run:
+          name: Upload << parameters.path >> coverage report to Codecov
+          command: bash ~/amplify-ios/.circleci/codecov.sh # From https://codecov.io/bash 
```

Comment:
Same question with this [one](https://github.com/aws-amplify/amplify-ios/pull/720/files#r488196045)

Reply:
Yes